apps/tlon-mobile/src/screens/ChannelScreen.tsx

```diff
@@ -81,7 +81,7 @@ export default function ChannelScreen(props: ChannelScreenProps) {
         }
       : {
           mode: 'newest',
-          firstPageCount: 10,
+          firstPageCount: 50,
```

Comment:
maybe bump around as well? Unless you found it noticeably impacted response time. Iirc post count was less important on average than you'd think

Reply:
With `around`, it's more about layout than anything else. In order to correctly position the anchored post on screen, we need the anchored post plus a few posts around it to have been rendered. The low initial count allows use to render exactly what we want on first render + scroll to it right away.